metadata/string/error.maxSize.multiple.json

```diff
@@ -0,0 +1,7 @@
+{
+  "_id": "error.maxSize.multiple",
+  "_type": "string.error",
+  "description": "Uploaded file is too large",
+  "value": "[% error.fileupload.invalid.too-large.multiple %]",
+  "value:cy": "[% error.fileupload.invalid.too-large.multiple %]"
```

Comment:
I think we can remove the `"value:cy"` lines, now

(The mechanism should default to `"value"` if `"value:cy"` isn't present, and both fields have the same value, now)

Reply:
i have now removed these